dj/sql/parsing/ast.py

```diff
@@ -539,14 +549,74 @@ def __str__(self) -> str:
     END)"""
 
 
+@dataclass(eq=False)
+class In(Expression):
+    """an in expression"""
```

Comment:
nit: Doc string formatting
```
    """
    An in expression
    """
```
Same in a couple other places.

Reply:
did all of them